docs/examples/add-to-your-application/ruby/Gemfile

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
+
+gem "oso-oso", path: "../../../../languages/ruby/"
```

Comment:
Make this `gem "oso-oso"` before merging

Reply:
Why? I intentionally want this to use the ruby code from the repo when being tested.